.clang-format

```diff
@@ -10,8 +10,8 @@ AlignEscapedNewlinesLeft: false
 AlignOperands: false
 # Purdy
 AlignTrailingComments: true
-AlignConsecutiveAssignments: true
-AlignConsecutiveDeclarations: true
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
```

Comment:
feel free to change these back, but this made for some pretty ugly code imo

Reply:
Yeah feel free to change any clang settings as you see fit -- I have stronger opinions about the eslint style settings because I've curated that a bit more, but these clang-format settings are a default I pulled somewhere from the internet.